gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri May 22 11:23:16 CEST 2020
+#Wed Jul 29 16:46:00 CEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```

Comment:
For build speed reasons, can you change this back to `-bin`? That results in a smaller download and speeds up our builds a little bit on CI. We don't really need the Gradle sources.

Reply:
I found it easier to use with code completion and documentation in gradle files. Will revert to -bin